modules/prebidServerBidAdapter/index.js

```diff
@@ -133,6 +133,13 @@ function formatUrlParams(option) {
       let temp = option[prop];
       option[prop] = { p1Consent: temp, noP1Consent: temp };
     }
+    if (utils.isPlainObject(option[prop]) && (!option[prop].p1Consent || !option[prop].noP1Consent)) {
+      ['p1Consent', 'noP1Consent'].forEach((conUrl) => {
+        if (!option[prop][conUrl]) {
+          utils.logWarn(`s2sConfig.${prop}.${conUrl} was not defined in manually set s2sConfig.  This may prevent PBS requests from executing in certain cases.  Please define field manually or instead use the defaultVendor settings of host company (if avialable).`);
```

Comment:
I've been encouraging folks to keep messages short because they don't minify.

How about something more terse like "s2sConfig.${prop}.${conUrl} not defined. PBS request will be skipped in some P1 scenarios."

Reply:
thanks, makes sense.  updated.